src/herder/TxQueueLimiter.h

```diff
@@ -73,10 +36,6 @@ class TxQueueLimiter
     // limits.
     std::shared_ptr<SurgePricingLaneConfig> mSurgePricingLaneConfig;
 
-    // Quick lookup of relevant account IDs, needed temporary to maintain
```

Comment:
The comment says this can be removed when the stacks are removed, but now where are we actually enforcing 1 tx per account in tx limiter? I think this was a valid sanity check (that helped catch tx queue bugs)

Reply:
i think it was a somewhat useful sanity check, but i don't really feel strongly. i'm ok with removing it.